models/optional_modules/consent/snowplow_web_consent_cmp_stats.sql

```diff
@@ -5,6 +5,53 @@
   )
 }}
 
+{%- if target.type in ('postgres') -%}
+
+with events as (
+
+  select
+    event_id,
+    domain_userid,
+    page_view_id,
+    domain_sessionid,
+    derived_tstamp,
+    event_name,
+    event_type,
+    cmp_load_time,
+    -- postgres does not allow the IGNORE NULL clause within last_value(), below workaround should do the same: removing NULLS using array_remove then using the COUNT window function (which counts the number of non-null items) to access the array using that as its index position
```

Comment:
Can you for my own sanity put in this comment that count is bounded up to the current row as well please, you may remember my decent in to madness before I worked that out :)